packages/fxa-js-client/client/FxAccountClient.js

```diff
@@ -2838,6 +2838,67 @@ FxAccountClient.prototype.createSupportTicket = function (
   });
 };
 
+/**
+ * Update a user's ecosystem anon ID
+ *
+ * @method updateEcosystemAnonId
+ * @param {String} sessionToken sessionToken obtained from signIn
+ * @param {String} ecosystemAnonId the new Ecosystem Anonymous ID
+ * @param {Object} [options={}]
+ *   @param {String} [options.ifNoneMatch]
+ *   Set the If-None-Match header to the specified value.
+ *   Use '*' to only update ID if one is not already set.
+ *   @param {String} [options.ifMatch]
+ *   Set the If-Match header to the specified value.
+ *   Use the first part of an existing ID to only update
+ *   the ID if the values do not fuzzy match.
+ * @return {Promise} A promise that will be fulfilled with an empty response object
+ */
+FxAccountClient.prototype.updateEcosystemAnonId = function (
```

Comment:
I take it this part will get removed in Danny's PR?

Reply:
Mmm ya, I guess I can remove all this.